PennMobile/src/main/java/com/pennapps/labs/pennmobile/RegistrarTab.java

```diff
@@ -72,11 +88,10 @@ public void onListItemClick(ListView l, View v, int position, long id) {
         args.putBoolean(getString(R.string.registrar_search), favorites);
         fragment.setArguments(args);
 
-        FragmentManager fragmentManager = getActivity().getSupportFragmentManager();
+        FragmentManager fragmentManager = mActivity.getSupportFragmentManager();
         fragmentManager.beginTransaction()
-                .replace(frameID, fragment)
+                .add(frameID, fragment)
```

Comment:
When the back button is pressed, the fragment manager removes the fragment, and so if we use replace, the old one would be gone as well.